src/Microsoft.OData.Edm/Vocabularies/AuthorizationVocabularies.xml

```diff
@@ -115,39 +111,39 @@
         </Property>
         <Property Name="RefreshUrl" Type="Edm.String" Nullable="true">
           <Annotation Term="Core.Description" String="Refresh Url" />
-          <Annotation Term="Core.IsURL" Bool="true"/>

```

Comment:
>Bool="true" [](http://example.com/codeflow?start=40&length=11)

I remember I fixed the ODL to accept omitting 'Bool="true" for tag. Can you double confirm that?

Reply:
@xuzhg We read the value optionally https://github.com/OData/odata.net/blob/d2108632a71a780998c25f05e536826972c1eb61/src/Microsoft.OData.Edm/Csdl/Parsing/CsdlDocumentParserBase.cs#L617